cybersecuritytools/csls/generate_cloudwatch_logs/put_cloudwatch_logs.py

```diff
@@ -0,0 +1,157 @@
+"""Module for generating test data of various types, send data to Cloudwatch."""
+import json
+from dataclasses import dataclass
+from datetime import datetime, timezone
+from functools import lru_cache
+from typing import Any, Dict, List
+from uuid import uuid4
+
+import boto3
+import mypy_boto3_logs as logs
+from mypy_boto3_logs.type_defs import PutLogEventsResponseTypeDef
+
+
+class LogEventResponseReal(PutLogEventsResponseTypeDef):
+    ResponseMetadata: Dict[Any, Any]
+
+
+@lru_cache(maxsize=None)
+def logs_client() -> logs.CloudWatchLogsClient:
+    return boto3.client("logs")
```

Comment:
Look, I'm always happy to see a memoized function, but I think this could be more simply be expressed as a top-level constant?

```suggestion
logs_client: logs.CloudWatchLogsClient = boto3.client("logs")
```

Reply:
I've removed the `lru_cache` decorator completely. It's extra complexity for an unnecessary optimisation.